.eslintrc.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     "es6": true,
     "jquery": true
   },
+  parserOptions: {
+    "ecmaVersion": "latest"
```

Comment:
I know this doesn't actually change anything since `latest` has been the default, but should we actually use something like `2020` to align with `esversion` with jshint?